pennylane/interfaces/tf.py

```diff
@@ -44,25 +40,25 @@
 
 The :ref:`QNode decorator <qnode_decorator>` is the recommended way for creating QNodes
 in PennyLane. The only change required to construct a TensorFlow-capable QNode is to
-specify the ``interface='tfe'`` keyword argument:
+specify the ``interface='tf'`` keyword argument:
 
 .. code-block:: python
 
     dev = qml.device('default.qubit', wires=2)
 
-    @qml.qnode(dev, interface='tfe')
+    @qml.qnode(dev, interface='tf')
     def circuit(phi, theta):
         qml.RX(phi[0], wires=0)
         qml.RY(phi[1], wires=1)
         qml.CNOT(wires=[0, 1])
         qml.PhaseShift(theta, wires=0)
         return qml.expval(qml.PauliZ(0)), qml.expval(qml.Hadamard(1))
 
-The QNode ``circuit()`` is now a TensorFlow-capable QNode, accepting ``tfe.Variable`` objects
+The QNode ``circuit()`` is now a TensorFlow-capable QNode, accepting ``Variable`` objects
```

Comment:
I'm unsure the best way to write this so that it is clear. The import paths are different depending on the version of TensorFlow:

* 1.X: `tensorflow.contrib.eager.Variable`
* 2.X: `tensorflow.Variable`

I thought dropping the full path would make it clear, it's up to the user to look up the tf docs to find out where Variable is.